src/sidebar/components/test/top-bar-test.js

```diff
@@ -17,7 +17,11 @@ describe('topBar', function() {
         bindings: require('../login-control').bindings,
       })
       .component('searchInput', {
-        bindings: require('../search-input').bindings,
```

Comment:
Now that this component is no longer an Angular component, it no longer exports an object with a `bindings` property. I've just reproduced the previous value directly in the test.

Reply:
Thanks for the heads up. I still find some of the bindings stuff during conversion to be a mind-melter :)